packages/syft/src/syft/core/tensor/autodp/phi_tensor.py

```diff
@@ -522,6 +522,23 @@ def sum(
         """
         return self.gamma.sum(*args, **kwargs)
 
+    def mean(
+        self,
+        *args: Tuple[Any, ...],
+        **kwargs: Any,
+    ) -> Union[
+        TensorWrappedPhiTensorPointer, MPCTensor, TensorWrappedGammaTensorPointer
+    ]:
+        """Apply the "mean" operation to "self"
+
+        Args:
+            y Tuple[Any, ...] : second operand.
+
+        Returns:
+            Union[TensorWrappedPhiTensorPointer,MPCTensor] : Result of the operation.
+        """
+        return self.gamma.mean(*args, **kwargs)
```

Comment:
When calling `mean()` on a PhiTensor, the output should still be a PhiTensor, since no new data subjects were introduced.

Reply:
@IshanMi  maybe a scenario like if the individual data points belonged to different data subjects, I think at that point it would be become a gamma tensor?

Ex:
<pre><code>
data_subjects = np.array( [DSA("GrandMaster"),DSA("Ishan")])

data_subjects.mean() would be 
np.array(DSA({"GrandMaster","Ishan"}))
</pre></code>

I think we could do like
<pre><code>
if len(self.data_sujects.sum()) ==1:  # to check the uniqueness.
      #create phiTensor pointer
else:
      create gamma tensor pointer.


</pre></code>

Reply:
Reminder before Merge :)